server/routes/brainImaging.ts

```diff
@@ -18,6 +20,15 @@ export const api: any = {
 			response: {
 				typeId: 'GetBrainImagingResponse'
 			}
+		},
+		post: {
```

Comment:
combine replace get{} and post{} with all{}

Reply:
@xzhou82 I told Irene to declare `post` separated. As part of refactoring the routes code, I found out that it's not good to use `all`, it results in 20+ route methods being set besides `get` and `post`. For example, if I console.log the route objects from the `express` app, I see `head`, `patch`, `delete`, and many other methods being set, when we only want to set `get` and `post`.